omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -22,11 +22,6 @@
 
 source git: 'https://github.com/DataDog/integrations-core.git'
 
-gcc_version = ENV['GCC_VERSION']
-if gcc_version.nil? || gcc_version.empty?
-  gcc_version = '10.4.0'
-end
```

Comment:
Just to be sure I understood. It means that we are going to use the default gcc binary shipped with the build image, so the version of gcc used to build the python dependencies might differ depending on it. Am I right? Is this desirable? We got some issues in the past because we were using different versions of gcc.

Reply:
That's correct. So far that's:
- 11 for RPM
- 5 for SUSE
- 9 for Debian/Ubuntu

All version appear to build the agent fine (the only issue I found was prior to version 5 which lacked some builtin for overflow safe arithmetic)

Do you remember what was the problem with different GCC versions?

Reply:
The version used in Centos 6 was too old and some flags that were needed to build some wheels did not exist. It's no longer a problem since we do not support it anymore. If using different versions is fine for you then that's all good!

Reply:
Ah yes I've seen some flags being sprinkled to remove some warnings. I also had some issue with libxmlsec which doesn't check if the compiler supports the flags it adds, but the extra warnings can be disabled altogether :relieved: